docs/build/windows.md

```diff
@@ -176,7 +176,7 @@ You can use Windows Settings to add these environment variables permanently:
 
 **Requirements**:
 * emscripten 2.0.23+
-* node.js 14+
+* node.js 14.6+
```

Comment:
Noting that update in the dev-environment in-repo docs.

Though, there's no Node.js entry on the Linux one... hmm.

Reply:
We're going to node 18 hopefully.